components/PostCard.tsx

```diff
@@ -75,27 +73,24 @@ const PostCard = ({ post }: Props) => {
               fill
               sizes="100vw"
               style={{
-                filter:
-                  isPostADraft(post) || isPostInTheFuture(post)
-                    ? "grayscale(50%)"
-                    : "none",
-
-                objectFit: "cover"
-              }} />
+                filter: isUpcomingPost(post) ? "grayscale(50%)" : "none",
 
+                objectFit: "cover",
+              }}
+            />
           </Link>
           <div className="tags">
             {post.tags.map((tag) => (
-              (<Link
+              <Link
                 href="/"
                 key={tag}
                 onClick={(event) => {
                   event.preventDefault();
                   dispatch(setTags([tag]));
-                }}>
+                }}
+              >
                 #{tag}
-
-              </Link>)
+              </Link>
```

Comment:
What about this? Will we leave this format like this?

Reply:
Discussed yesterday, there's no issue here